client/modules/Slack/SlackBotSender.php

```diff
@@ -41,8 +41,8 @@ class SlackBotSender
      */
     public function __construct($slackBotUrl, $auth = null)
     {
-        $this->slackBotUrl = $slackBotUrl;
-        $this->auth        = $auth;
+        self::$slackBotUrl = $slackBotUrl;
```

Comment:
@lv-dev so we can not have two instance of slackBotSender in process, i mean two client to to different slack bots.